qp_shogun/sortmerna/databases/rRNA_databases/scripts/generate_databases.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+rootdir="/home/skarthikeyan/ARB_SILVA_databases"
```

Comment:
perhaps remove this folder and add a comment to add your own?

Reply:
Done.